styles.css

```diff
@@ -1,785 +1,862 @@
+        background-repeat: no-repeat;
+        background-position: 0.5vw 2vw;
+    }
+    .mainheader{
+        width: 30%;
+        height: 3.25vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 3.5vw;
+        line-height: 4.25vw;
+        display: flex;
+        align-items: center;
+        letter-spacing: 0.37px;
+        color: #091E42;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        padding-left: 15vw;
+        padding-top: 20vw;
+    }
+    .mainpara{
+        width: 68%;
+        height: 7.5vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 1.65vw;
+        line-height: 3.5vw;
+        color: #344563;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        padding-left: 15vw;
+    }
+    .let{
+        width: 50%;
+        height: 1vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 2.08vw;
+        line-height: 3.5vw;
+        color: #7F8CFF;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        padding-left: 10vw;
+    }
+    .mainpagelink{
+        width: 40vw;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        order: 2;
+        flex-grow: 0;
+    }
+    .bottom{
+      width: 68vw;
+      height: 20vw;
+      margin-left: 10vw;
+      background-image: url("bottoback.png");
+      background-size: 68vw 15v;
+      background-repeat: no-repeat;
+      order: 2;
+      flex-grow: 0;
+    }
+    .can{
+        display: flex;
+        flex-grow: 0;
+        width: 50%;
+        height: 1vw;
+        margin-left: 3.5vw;
+        }
+    .card{
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 16px;
+        gap: 12px;
+        width: 95vw;
+        height: 100%;
+        background: #FFFFFF;
+        border: 1px solid #DFE1E6;
+        border-radius: 16px;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 1vw;
+        margin-right: 1vw;
+    }
+        .partone{
+            width: 50vw;
+            height: 100%;
+            background-image: url("Portfolio.png");
+            background-size: 45vw 100%;
+            background-repeat: no-repeat;
+        }
+        .parttwo{
+            display: flex;
+            flex-direction: column;
+            width: 50vw;
+            height: 100%;
+        }
+        .tonic{
+            width: 45vw;
+            height: 0.5vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 700;
+            font-size: 5.208vw;
+            line-height: 5vw;
+            color: #091E42;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            margin-left: 2.5vw;
+        }

```

Comment:
- [ ] Please make sure the font size of the texts in the project card looks similar to the Figma design and that they're also aligned properly as I noticed in your design the texts are a bit larger and misaligned. It's important to pay attention to small details like this as a frontend developer as they add a more polished look to your UI.
- Figma design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/ac19aadf-8cee-4dca-b8b0-0d350b9383aa)
- Your design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/1b95dfb9-7b3f-4b85-9f96-15ecc8e807ac)